odl/contrib/pytorch/examples/basic.py

```diff
@@ -0,0 +1,45 @@
+"""Demonstration of basic ODL->pytorch integration functionality."""
+
+import numpy as np
+import torch
+import odl
+from odl.contrib.pytorch import TorchOperator
+
+# Define ODL operator
+matrix = np.array([[1, 2, 3],
+                   [4, 5, 6]], dtype='float32')
+matrix_op = odl.MatrixOperator(matrix)
+
+# Wrap as torch operator
+torch_op = TorchOperator(matrix_op)
+
+# Define evaluation point and wrap into a variable, mark as requiring gradient
+x = torch.ones((3,)).type(torch.FloatTensor)
```

Comment:
There's no `dtype` argument we can use?

Reply:
Haven't seen one so far. They have some work to do on the convenience side. This is the way to produce a certain dtype that I've seen everywhere in their docs. Seems wasteful, but maybe the JIT will take care of that.